### ISSUE TITLE
Having difficult in making modal dialog work

### ISSUE THREAD

**Issue by xAngular** [angular/angular#7210]
I can't find documentation or sample code how to do modal dialog in Angular2. Then I tried to follow the way in Angular1 to make modal dialog work. Here are two plunkers I came up. But neither work. Can someone help me take a look and see what are wrong.
http://plnkr.co/edit/ikIt7YGDSCWqdHV0GHWw?p=preview
http://plnkr.co/edit/ZzWehRSxj3TQeLApO44K?p=preview

**Comment by garethdn** [comment]
[This link ](http://stackoverflow.com/a/34748485/1128290) might help you out

**Comment by xAngular** [comment]
@garethdn, I appreciate your help so much. I am new to Angular2. I checked the link you provided, however, I am still not very clear how to make my modal dialog work use the idea in the link. Is a modal dialog a dynamic component? All I want is to popup the modal dialog when the "Add Hero" button is clicked. It is such a very basic feature for an application. I am very surprised that I can't find any documentation or sample code available. I feels like I am in nightmare. Will it be possible for you to make some changes to the plunkers provided above to make it work?

**Comment by ericmartinezr** [comment]
>I am very surprised that I can't find any documentation or sample code available

[Uhmmmmm](http://lmgtfy.com/?q=angular2+modal)

**Comment by xAngular** [comment]
@ericmartinezr , the search result comes back from yours is better than mine "AngularJS 2.0 modal". I will try to follow the links you provided.

**Comment by robwormald** [comment]
@garethdn and @ericmartinezr thanks for jumping in. 

@xAngular i'm going to close this issue as its not an issue with the framework itself - in the future please keep Github Issues only for reporting bugs or problems. You'll find lots of helpful people in our [gitter](https://gitter.im/angular/angular) channel for these kinds of how-to questions. Thanks!

**Comment by xAngular** [comment]
I asked for help at the gitter channel first. No body gave feedback, then I went to Github for help. Even though it is not an issue of the framework, but I do think it is a feature badly needed. Apparently, Angular1 has done this better. Of course, it is only my humble opinion as a beginner.

**Comment by xAngular** [comment]
I played around a little bit with the bootstrap modal dialog. It turns out that I can combine the bootstrap modal dialog with Angular 2 form to achieve what I want without using any libraries particularly developed for modal in Angular 2. All I need is to include the jQuery and bootstrap libraries in index.html. I have updated the plunders I originally created when I asked for help. Here are the new plunkers in case some other developers new to Angular 2 like myself need them:
http://plnkr.co/edit/OoGL33GfEuzS8sIVuNcn?p=preview
http://plnkr.co/edit/B6x7B9m2I7AQLsq6bceu?p=preview
